src/api/schemas/equipments.py

```diff
@@ -0,0 +1,28 @@
+import re
+from typing import List
+
+from pydantic import BaseModel, validator
+
+from src.api.schemas.parts import Part
+
+
+class EquipmentBase(BaseModel):
+    name: str
+
+    @validator('name')
+    def name_match(cls, name):
+        if not re.match(r'^[\w\s.-]+$', name):
+            raise ValueError(f"Name '{name}' is incorrect")
+        return name.lower()
+
+
+class EquipmentCreate(EquipmentBase):
+    pass
```

Comment:
It's not clear for me, why you've made empty class?

Reply:
Removed all empty classes.